justfile.fedimint.just

```diff
@@ -236,11 +236,10 @@ docs: build-docs
     $open_cmd "$fallback_index_path" || true
   fi
 
-release-bump-version VERSION="prerelease":
-  cargo workspaces version --all --exact --no-git-commit --yes --force '*' --pre-id rc {{VERSION}}
-
-release-bump-version-custom VERSION:
-  cargo workspaces version --all --exact --no-git-commit --yes --force '*' custom {{VERSION}}
+# Bump all versions
+# ex: `just bump-version 0.5.0-alpha 0.5.0-rc.0`
+bump-version FROM_VERSION TO_VERSION:
+  find . -name 'Cargo.toml' -exec sed -i 's/{{FROM_VERSION}}/{{TO_VERSION}}/g' {} +
```

Comment:
what if it changes version of some other dependency :/

Reply:
The build will fail and we'll see it.

Since we're not using dependencies in versions `-alpha` or `-rc.x` a lot, it's theoretically a problem, but in practice good enough.

Reply:
right we now use -alpha.0 on master.

Reply:
> right we now use -alpha.0 on master

I think we're just `-alpha` on master

> what if it changes version of some other dependency :/

Yeah, it may still be a bit of a mess since we might need to manually inspect failures if the build fails (quick grep for 0.4.1 and 0.4.2 shows we'll bump into this soon).

The ideal solution is to update everything to workspace = true so we only need to update the root Cargo.toml when we do a version bump.